caracara/common/pagination.py

```diff
@@ -49,8 +56,13 @@
     }
 }
 
-Style 3
-Used in Intel. TBC.
+
+Style 3 (Token After)
+This style works exactly the same as Style 2, except the token is stored in
+an attribute named `after`.
+
+Style 4 (Token _marker)
+Used in Intel. TBC.      (╯°□°)╯︵ ┻━┻
```

Comment:
Yup.  I'm actually leaning towards "threats" (Intel, Malquery, recon) as being our next area of focus.  This should flush out pretty quickly at that point.

(Works similar as an after token, except you consume down to a total of zero.)

Reply:
I documented it in the version I pushed. I still called it the "Intel" style, but we can rework that once we get to implementing those APIs. It's just a set of comments anyway, rather than anything functional.